demo/components/LiveEdit.js

```diff
@@ -17,21 +17,13 @@ const StyledProvider = styled(LiveProvider)`
   margin-bottom: ${polished.rem(100)};
 `
 
-const LiveWrapper = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: stretch;
-  align-items: stretch;
-
-  @media (max-width: 600px) {
-    flex-direction: column;
-  }
-`
 
 const column = css`
   flex-basis: 50%;
   width: 50%;
   max-width: 50%;
+  display:inline;
```

Comment:
Can you remove the non-flexbox CSS related code? This isn't clean, and probably not a solution to the problem you were trying to solve ;)